src/app/components/Transactions/index.tsx

```diff
@@ -59,8 +62,16 @@ export const Transactions: FC<TransactionsProps> = ({
   verbose = true,
 }) => {
   const { t } = useTranslation()
+  // We only want to show encryption status of we are listing transactions
+  // from paratimes that actually support encrypting transactions
+  const canHaveEncryption = doesAnyOfTheseLayersSupportEncryptedTransactions(
+    transactions?.map(tx => tx.layer),
+  )
   const tableColumns = [
     { content: t('common.status') },
+    ...(canHaveEncryption
+      ? [{ content: (<LockIcon htmlColor={COLORS.grayMedium} />) as unknown as string }]
+      : []), // The table does support widgets in the column headers, but the TS definition is unaware of that.
```

Comment:
Table does not support it too well
https://github.com/oasisprotocol/explorer/blob/a77aa3ac81aa59ba4dd532f2fb4db90161052ce9/src/app/components/Table/index.tsx#L108

Reply:
Yes, that could be improved. (By generating the key using the index, or something similar.)